networkdb/delegate.go

```diff
@@ -53,6 +53,7 @@ func (nDB *NetworkDB) handleNetworkEvent(nEvent *NetworkEvent) bool {
 			n.leaveTime = time.Now()
 		}
 
+		nDB.addNetworkNode(nEvent.NetworkID, nEvent.NodeName)
```

Comment:
Yeah it is already happening here: https://github.com/docker/libnetwork/pull/1406/files#diff-ba1627e58f1ba78407292538370e24f3R51